modules/odc-adding-components-to-an-existing-project.adoc

```diff
@@ -0,0 +1,19 @@
+// Module included in the following assemblies:
+//
+// applications/application_life_cycle_management/odc-viewing-application-composition-using-topology-view.adoc
+
+[id="odc-adding-components-to-an-existing-project_{context}"]
+= Adding components to an existing project
+
+.Procedure
+
+. Click on the *Add to Project* (image:odc_add_to_project.png[title="Add to Project"]) or press kbd:[Ctrl+Space]
```

Comment:
```suggestion
. Click the *Add to Project* (image:odc_add_to_project.png[title="Add to Project"]) or press kbd:[Ctrl+Space]
```
IBM SG: Use "click", not "click on", to refer to the action of selecting something on a graphical interface.